enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/EnterpriseAuthSubject.java

```diff
@@ -115,27 +105,27 @@ public boolean allowsSchemaWrites()
     @Override
     public String name()
     {
-        return subject.getPrincipal().toString();
+        return shiroSubject.getPrincipal().toString();
     }
 
-    Subject getSubject()
+    ShiroSubject getShiroSubject()
     {
-        return subject;
+        return shiroSubject;
     }
 
     private AccessMode.Static getAccessMode()
```

Comment:
Yes, we have discussed this before. Added checklist-item to the cleanup card to track it.